BICI_Psoc_workspace/Fingertip_V3_acc.cydsn/main.c

```diff
@@ -88,6 +169,17 @@ int main(void)
                     i2cReadBuffer[i+59] = CapSense_dsRam.snsList.row11[i].raw[0]; 
                     i2cReadBuffer[i+63] = CapSense_dsRam.snsList.row12[i].raw[0]; 
                 }
+                for(unsigned int i=0; i<1; ++i) // Proxi
+                {
+                    i2cReadBuffer[i+67] = CapSense_dsRam.snsList.proxi[i].raw[0]; 
+                } 
+                
+                if(TRANSFER_CMPLT == readAccel()){
+                    
+                    i2cReadBuffer[68] = X_out;

```

Comment:
i2cReadBuffer is uint16.... You will be missing 16bits. Did you test this???

Reply:
Also, you are rewriting over your proxi value that you just entered right before...

Reply:
I can't see the changes until you update your pull request

Reply:
The proxi thing was my misunderstanding, my mistake